Dockerfile

```diff
@@ -0,0 +1,29 @@
+ARG NODE_VERSION=20
+ARG OS_VERSION=bullseye-slim
+# Content of this file derived from 
+# https://github.com/nodejs/docker-node/blob/main/docs/BestPractices.md
+FROM node:${NODE_VERSION}-${OS_VERSION}
+
+# Build/Run time args
+ARG APP_DIR=mDL-Testing
+ARG NODE_USER=node
+ARG NODE_HOME=/home/${NODE_USER}
+
+# TODO: this currently prevent npm run build to work.
+#ENV NODE_ENV=production
+
+COPY . ${NODE_HOME}/${APP_DIR}
+WORKDIR ${NODE_HOME}/${APP_DIR}
+RUN npm clean-install --force && \
```

Comment:
By adding `--force` (which the README also suggests to get the local environment set up), it does disable some of npm's protections. It would be good to understand why that's the case.

Reply:
@NimeshNB could you please advise on this ?

Reply:
when we use ```npm install``` then we are getting unmet dependency error from ```react-qr-reader``` because react version used in this project is latest ```18``` while library has ```17```  thus we are doing ```npm install --force```